src/Stryker.Core/Stryker.Core.UnitTest/StrykerOptionsTests.cs

```diff
@@ -19,7 +19,7 @@ public class StrykerOptionsTests
         [InlineData("trace", LogEventLevel.Verbose)]
         public void Constructor_WithCorrectLoglevelArgument_ShouldAssignCorrectLogLevel(string argValue, LogEventLevel expectedLogLevel)
         {
-            var options = new StrykerOptions("c:/test", "Console", "", 0, null, argValue, false, 1, 80, 60, 0);
+            var options = new StrykerOptions(logLevel:argValue);
```

Comment:
Basepath change, as above, not sure if it matters but if it wasn't an intentional change..

Reply:
According Test name seems that only log level meters here.